_activities/pyhep.md

```diff
@@ -0,0 +1,45 @@
+---
+title: PyHEP Workshops
+author: "Eduardo Rodrigues"
+layout: default
+---
+
+# PyHEP "Python in HEP" Series of Workshops
+
+The **PyHEP workshops** are a new series of workshops initiated and supported by the HSF
+with the aim to provide an environment to discuss and promote the usage of Python in the HEP community at large.
+
+
+## PyHEP 2018
+
+The first workshop, **PyHEP 2018**, will be held as a pre-CHEP event in Sofia, Bulgaria, on 7-8 July 2018, just before the start of the [CHEP 2018 conference](http://chep2018.org/). It will focus on a review of where and how Python is used in our community, and what the future will hold.
```

Comment:
I would also put the Indico link here too (maybe for "first workshop").

Reply:
Oops, you're right! I'm adding the link to the word "agenda" on the following sentence.